src/Ch7/EmployeeStatic.java

```diff
@@ -0,0 +1,41 @@
+//Demonstrate Non-Access Type 'Static'
+/**
+ * author Naresh P
+ * since 2018
+ * version 1.1
+ */
+package Ch7;
+
+class employee{
+	
+	int id;

```

Comment:
It is better to use 'private' for all the instance variables, and have getter and setter methods.

for example:
private int id;

public int getId() {
     return id;
}

public void setId(int id) {
    this.id = id;
}